src/elm/arithmetic.js

```diff
@@ -75,6 +80,8 @@ class Multiply extends Expression {
     const product = args.reduce((x, y) => {
       if (x.isQuantity || y.isQuantity) {
         return doMultiplication(x, y);
+      } else if (x.isUncertainty && y.isUncertainty) {
+        return new Uncertainty(x.low * y.low, x.high * y.high);
```

Comment:
What do you think about divide?  I guess that would work a little different -- because the same approach does not apply. For example, `Uncertainty(5,10) / Uncertainty(5,10)` is definitely _not_ `Uncertainty(5/5, 10/10)` -- because `Uncertainty(1, 1)` is `1`.  But I think if we think about it from lowest possible value to highest possible value, then maybe it would be `Uncertainty(5/10, 10/5)`?  Or in otherwords, `Uncertainty A / Uncertainty B` is `Uncertainty(A.low/B.high, A.high/B.low)`? Does that make sense?  Should we implement it?